src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+  @Override
+  public void doWork() {
+    System.out.println("Excavator doWork");
```

Comment:
Your string is kind of debug message. Like something was happened.
Use more convenient way. For instance, "Excavator start working";